hassio/dock/addon.py

```diff
@@ -100,7 +109,8 @@ def _run(self):
                 ports=self.addons_data.get_ports(self.addon),
                 devices=self.addons_data.get_devices(self.addon),
                 environment=self.environment,
-                volumes=self.volumes
+                volumes=self.volumes,
+                tmpfs=self.tmpfs
```

Comment:
My push was rejected, I've tried it moments after your commit. I'm ok with the code as it is now, but if you prefer I can ditch the property and have a list in the options instead of a string.